### DISCUSSION TITLE
Bundle Identifier never Loads

### DISCUSSION THREAD

**Discussion by fajusto** [codemagic-ci-cd/codemagic-docs#2039] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1 . Open project
2. Go to edit workflow
3. Open Distribution pane
4. Set the App Store Connect Key
5. Then 'Provisioning profile type' to App Store
6. Should choose the Bundle Identier according the project.

### Expected results

Should load all Bundle Identiers to choose the one for this project.

### Actual results

Never loads.

![2023-09-12 14_21_32-Application settings - Codemagic](https://github.com/codemagic-ci-cd/codemagic-docs/assets/14909648/491143c8-c287-4f42-9f8b-50c29ba74450)

**Comment by hossam20520** [comment]
i have the same issue

**Comment by blackpatton17** [comment]
Same issue.

**Comment by eventonz** [comment]
Same issue also

**Comment by icarusdust** [comment]
Hi @fajusto @hossam20520 @eventonz @blackpatton17 , it should be alright now. Refresh the page and try again, let me know if it is still persisting.

**Comment by fajusto** [comment]
It is fixed, thanks!
